packages/generator/src/edmx-to-vdm/edmx-to-vdm-util.ts

```diff
@@ -155,3 +155,16 @@ export function typesForCollection(
 
 export const propertyJsType = (type: string): string | undefined =>
   type.startsWith('Edm.') ? edmToTsType(type) : undefined;
+
+export function hasUnsupportedParameterTypes(parameters: EdmxParameter[]) {
```

Comment:
The name of the import would actually be nice in the log message. Maybe we should pass the whole function / action import here?